### PR TITLE
less or equal to strictly less flavor size

### PR DESCRIPTION
**Pull request by alexv-ds**
## Changelog

:cl:
fix: with a flavor length of FLAVOR_TEXT_CHAR_REQUIREMENT characters, you can now join the game 
/:cl:

### CODE REVIEW

**Comment by ArdentSpark** [comment]
Makes sense to me, the text implied that you needed a minimum of (150), but it looks like if the code saw you had (150) it still blocked you. A small fix but still good nonetheless

File: code/_onclick/hud/new_player.dm [hunk 1]
```diff
@@ -138,7 +138,7 @@
 	var/mob/dead/new_player/new_player = hud.mymob
 
 	// SKYRAT EDIT BEGIN
-	if(length_char(new_player.client.prefs.read_preference(/datum/preference/text/flavor_text)) <= FLAVOR_TEXT_CHAR_REQUIREMENT)
+	if(length_char(new_player.client.prefs.read_preference(/datum/preference/text/flavor_text)) < FLAVOR_TEXT_CHAR_REQUIREMENT)
```
Comment (tf-4) [inline code/_onclick/hud/new_player.dm]:
```suggestion
	if(length_char(new_player.client?.prefs?.read_preference(/datum/preference/text/flavor_text)) < FLAVOR_TEXT_CHAR_REQUIREMENT)
```

**Review by tf-4 — COMMENTED** [review]
May as well get this while you're here

**Comment by Zenitheevee** [comment]
Just add an additional period after the end of your flavor text, Add a bit of mystery, Add a u to color. Use "Tallness" instead of "Height"

Shitposting asside 👍🏼

**Review by Zonespace27 — APPROVED** [review]
Good catch from my PR

**Review by tf-4 — APPROVED** [review]

**Pull request merged**